src/functionalTest/resources/features/F-004 - Put Dictionary /S-004.8_Existing_Data.td.json

```diff
@@ -0,0 +1,28 @@
+{
+  "_guid_": "S-004.8_Existing_Data",
+  "_extends_": "Put_Dictionary_Base",
```

Comment:
If you change to `"_extends_": "Put_Dictionary_Base_Definition_Store"`, you can then delete line 14 ?

Reply:
I actually meant to delete that file so all I had to do was add one line to this one